### ISSUE TITLE
Editor: Permalinks: Missing URL Slug Field

### ISSUE THREAD

**Issue by carinapilar** [Automattic/wp-calypso#50421]
### Steps to reproduce the behavior

1. On a Simple site, create a new post
2. Click on the Permalink section on the post settings
3. Try to edit the post's slug

#### What I expected to happen
To see the _URL Slug_ field to be able to change it.

#### What actually happened
There's no _URL Slug_ field. It only shows the field once you save the post as a draft. This isn't happening when creating pages.

#### Browser / OS version
Chrome 88.0.4324.192
Mac OS X 10.15.7

#### Does this happen on simple or atomic sites or both?
It only happens on Simple sites, wasn't able to replicate on Atomic and WordPress.org sites.

#### Reproducibility
Consistent.

#### Screenshot
When creating the post, before saving:
![PermalinkSlug1](https://user-images.githubusercontent.com/16104522/108986703-2f302b00-7671-11eb-8f20-c7731f06700b.jpg)

After saving the draft:
![PermalinkSlug2](https://user-images.githubusercontent.com/16104522/108986712-30615800-7671-11eb-8d92-f55a90538d53.jpg)

Reported by the user on 3768342-zd-woothemes

**Comment by supernovia** [comment]
A user [brought up this issue in our forums](https://wordpress.com/forums/topic/creating-an-ending-for-a-posts-url/#post-3642588). I can't duplicate it for anything, but here's what they see (left) vs what I see (right):

They have tried two different browsers, too.  @carinapilar are you still seeing this in staging versions?

<img width="625" alt="Screen Shot 2021-03-02 at 2 09 09 PM" src="https://user-images.githubusercontent.com/3476361/109716315-f3f69600-7b61-11eb-8aeb-508013478b8f.png">

**Comment by carinapilar** [comment]
@supernovia, yes, I can still replicate the issue. Interesting, I can replicate on that user's account, when creating a new post.

**Comment by supernovia** [comment]
I can't change it _before_ the post is saved as a draft. After that I can, though. (On my own site)

Do you see the same thing on your own site? I wasn't able to replicate this on the user's account.

**Comment by supernovia** [comment]
From my test site:
<img width="1218" alt="Screen Shot 2021-03-02 at 3 20 39 PM" src="https://user-images.githubusercontent.com/3476361/109723280-1b9e2c00-7b6b-11eb-8feb-472b62021186.png">

**Comment by carinapilar** [comment]
> I can't change it before the post is saved as a draft. After that I can, though. (On my own site) Do you see the same thing on your own site? I wasn't able to replicate this on the user's account.

That's right, the issue only happens _before_ you save the post as a Draft. This doesn't happen with pages though, and also doesn't happen on Atomic and WordPress.org sites. This behavior is only happening when creating posts on a Simple site.

**Comment by annacmc** [comment]
I can replicate this: Tested this on a WPCOM simple site and could repro. I tested:

On a simple site post = did not have the slug editing field until post was saved as a draft
On a simple site page = Immediately had the slug editing field, before page was saved as a draft

However, I tested this on an Atomic testing site and a WordPress.org (localhost, not JP connected) site and experienced the same behavior there, too. So I am not sure if this is a WPCOM specific issue. I tried in a number of different environmen
ts and could not find one where I was able to edit the slug on a post prior to editing a draft - only with pages.

**Comment by carinapilar** [comment]
> However, I tested this on an Atomic testing site and a WordPress.org (localhost, not JP connected) site and experienced the same behavior there, too.

Hm, interesting, when I first tested and reported this, it wasn't possible to replicate on Atomic and WordPress.org sites. I wonder what changed.
